wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/theme/ajax/AjaxWRadioButtonSelectExample.java

```diff
@@ -51,7 +51,6 @@
 	public AjaxWRadioButtonSelectExample() {
 		setLayout(new FlowLayout(Alignment.VERTICAL, 0, 6));
 
-		rbSelect.setAccessibleText("Content selection");

```

Comment:
Remove this line but replace line 67 
`rbSelect.setToolTip("Choose one");` with
`rbSelect.setToolTip("Content selection");` and
add a TODO to fix the example which is particularly poor in its accessibility implementation.